bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -386,8 +386,8 @@ inj_plots = wflow.cp.get_subsections('pygrb_plot_injs_results')
 # from the set of requested injection plot types
 inj_plots = [inj_plot for inj_plot in inj_plots if inj_plot not in inj_sets]
 for inj_set in inj_sets:
-    inj_file = list(filter(lambda x: inj_set.lower() in x.lower(), inj_files))
-    inj_file = inj_file[0] if len(inj_file) == 1 else None
+    # Retrieve the injections result File corresponding to the inj_set label
+    inj_file = inj_files.find_output_with_tag(inj_set)[0]
```

Comment:
Perhaps a sanity check here:
```
inj_files = inj_files.find_output_with_tag(inj_set)
if not len(inj_files) == 1:
    raise ValueError("MESSAGE.")
inj_file = inj_files[0]
```

Reply:
On second thought: It would be good to add a kwarg, and this check, to the `find_output_with_tag` method. Something like a `fail_if_not_single_file=True` kwarg (which by default is False), and then move this check there and run it only if the flag is given.

Reply:
I just pushed a commit that should address this.